applications/style.css

```diff
@@ -364,16 +367,61 @@ body {
 
 .application-details .application-details-actions .application-details-reject {
   color: var(--red-color);
-  height: auto;
   background-color: white;
-  margin-left: auto;
-  margin-right: auto;
-  text-decoration: underline;
+  margin: 0 auto;
+  width: 40%;
+  border: 2px solid var(--red-color);
+}
+
+.application-details
+  .application-details-actions
+  .application-details-reject:hover,
+.application-details
+  .application-details-actions
+  .application-details-reject:active {
+  background: var(--red-color);
+  color: var(--white);
+  transition: transform 0.2s ease;
+}
```

Comment:
To simplify the CSS, use the :is() pseudo-class to group :hover and :active states. This avoids repeating the full selector:

```css

.application-details .application-details-actions .application-details-reject:is(:hover, :active) {
  background: var(--red-color);
  color: var(--white);
  transition: transform 0.2s ease;
}

```

Reply:
Thank you for the wonderful suggestion! I will surely do that.